.azure-pipelines/BranchProtectionForPrivateRepo.yml

```diff
@@ -2,7 +2,7 @@ jobs:
   - job: "BranchProtectionForPrivateRepo"
     pool:
       vmImage: 'Ubuntu 16.04'
-    condition: and(variables['PRIVATE'], eq(variables['System.PullRequest.TargetBranch'], 'master'))
+    condition: and(variables['PRIVATE'], eq(variables['System.PullRequest.TargetBranch'], 'main'))
```

Comment:
should protect both main & master initially?

Reply:
This is for block private spec repo. 
I am testing another task for blocking public spec repo master merge.

Reply:
already added check , after renaming, we can merge the PR and mark the check `public.rest-api-specs (ShouldSendPRToMain)` as required, e.g. https://github.com/Azure/azure-rest-api-specs/pull/15465/checks?check_run_id=3216862934